addons/missileguidance/functions/fnc_ahr_onFired.sqf

```diff
@@ -0,0 +1,71 @@
+#include "script_component.hpp"
+/*
+ * Author: Brandon (TCVM)
+ * Sets up SACLOS state arrays (called from missileGuidance's onFired).
+ *
+ * Arguments:
+ * Guidance Arg Array <ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call ace_missileguidance_fnc_SACLOS_onFired
+ *
+ * Public: No
+ */
+params ["_firedEH", "_launchParams", "", "", "_stateParams"];
+_firedEH params ["_shooter","","","","","","_projectile"];
+_stateParams params ["", "_seekerStateParams"];
+_launchParams params ["_target"];
```

Comment:
you are consistent, so it's not a problem but by default missile guidance puts shooter instead of target on first param
```
_launchParams params ["_shooter","_targetLaunchParams","_seekerType","_attackProfile","_lockMode","_laserInfo"];
_targetLaunchParams params ["_target", "_targetPos", "_launchPos"];
```